chips/earlgrey/src/aes.rs

```diff
@@ -149,6 +151,31 @@ impl<'a> Aes<'a> {
         self.registers.status.is_set(STATUS::IDLE)
     }
 
+    /// Must wait for IDLE, for trigger to set.
+    /// On reset, AES unit will first reseed the internal PRNGs
+    /// for register clearing and masking via EDN, and then
+    /// clear all key, IV and data registers with pseudo-random data.
+    /// Only after this sequence has finished, the unit becomes idle
+    ///
+    /// NOTE: This is necessary for Verilator
```

Comment:
Is there a harm in just doing this for both hardware and ventilator? It's a bounded loop waiting for startup, which seems pretty reasonable

Reply:
Probably not, might be a little faster on HW if the AES IP is not responsive (I guess?). Should we just add it to both then?

Reply:
Even when there is an OT ASIC, I still expect people to use Verilaotr. It's open RTL, I image people would be intested in using that . The implementation would be clearer without the extra config switches

Reply:
Anytime there is a wait loop it can be confusing: why was this added? is it needed all the time, or just on some chips? is this documented in the datasheet? if I remove this, what will break? Having the config switch provides that clarity, as it won't be included when it isn't needed, and makes it clear when it is.

Reply:
Wouldn't a comment just fix that?